cmd/cluster/support/post.go

```diff
@@ -89,13 +92,21 @@ func (o *postOptions) run() error {
 	// and load it into the LimitedSupport variable
 	readTemplate()
 
+	// Parse all the '-p' user flags
+	parseUserParameters()
+
 	// Check that the cluster key (name, identifier or external identifier) given by the user
 	// is reasonably safe so that there is no risk of SQL injection
 	err := ctlutil.IsValidClusterKey(o.clusterID)
 	if err != nil {
 		return err
 	}
 
+	// For every '-p' flag, replace it's related placeholder in the template
+	for k := range userParameterNames {
+		replaceFlags(userParameterNames[k], userParameterValues[k])
```

Comment:
To me, `replaceFlags` doesn't describe the function well enough, how about:

**suggestion:** Rename this function to `insertFlags`

Reply:
The reason behind the wording `replaceFlags` is that the template contains the variables such as `${CONFIGURATION}` and that is _replaced_ by the input given in the command. When you drill down, it uses `strings.ReplaceAll`.

Reply:
Would `replaceWithFlags` fit even better then? Semantically, `replaceFlags` would mean that we replace the flags, not that we replace the variables with flags (which is why I initially suggested `insertFlags`)

Reply:
`replaceWithFlags` sounds good.